### DISCUSSION TITLE
WWXYDT2

### DISCUSSION THREAD

**Discussion by AntiPontifex** [iptv-org/community#842] [Q&A]
### Channel ID

WWXYDT2.pr

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by BellezaEmporium** [comment]
WWXY is Puerto Rico, not US-related afaik

**Comment by AntiPontifex** [comment]
Puerto Rico is a US protectorate; so it is an American territory. Also, note the W in the call sign as proof.

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/issues/19802
